doc/nrf/app_dev/build_and_config_system/index.rst

```diff
@@ -204,18 +204,24 @@ Other output build files
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| :file:`zephyr.meta`               | A file with the Zephyr and nRF Connect SDK git hashes for the commits used to build the application.                              |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| :file:`tfm_s.elf`                 | An ELF file for the TF-M image that is being built. Can be used for debugging purposes.                                           |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| :file:`manifest.json`             | Output artifact that uses information from the :file:`zephyr.meta` output file.                                                   |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| :file:`dfu_multi_image.bin`       | Multi-image package that contains a CBOR manifest and a set of user-selected update images,                                       |
+|                                   | such as firmware images for different cores.                                                                                      |
+|                                   | Used for DFU purposes by :ref:`ug_matter` and :ref:`ug_zigbee` protocols.                                                         |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| :file:`signed_by_b0_s0_image.bin` | Intermediate file generated by :file:`sign_cmake` and used for :ref:`bootloader`.                                                 |                                                                                                                                                                                             |
+|                                   | Can be used to generate the :file:`signed_by_mcuboot_and_b0_s0_image_update.bin` when using NSIB with MCUboot.                    |
++-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| :file:`signed_by_b0_s1_image.bin` | Intermediate file generated by :file:`sign_cmake` and used for :ref:`bootloader`.                                                 |
+|                                   | Can be used to generate the :file:`signed_by_mcuboot_and_b0_s1_image_update.bin` when using NSIB with MCUboot.                    |
```

Comment:
```suggestion
| :file:`signed_by_b0_s0_image.bin` | Intermediate file only signed by NSIB.                                                 |                                                                                                                                                                                             |
|                                   |                    |
+-----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
| :file:`signed_by_b0_s1_image.bin` | Intermediate file only signed by NSIB.                                                 |
|                                   |                    |
```

I struggle to see the usefulness of describing these files here except that they're just intermediate files. Is it even possible to use NSIB without MCUboot? Would you have the application as S0/S1 then? Or would you need some other bootloader? This sounds very edge-case to me, but please correct me if there is an actual use-case for this.
The way I understood the question from the customer was why we had the two set of files and the difference between them. There was also some confusion in the case w.r.t. swapping of S0/S1 and how the mcuboot signature was being used. So I think it's more important to explain why the image needs to be signed by both NSIB and MCUboot.